process.lua

```diff
@@ -0,0 +1,6 @@
+local process = {}
```

Comment:
part of me wonders if the version info in main should be hoisted here? 🤔

Reply:
updated in [2362e37](https://github.com/ar-io/ar-io-network-process/pull/384/commits/2362e37ec2be9fa515aae8cc3d1936011b4e7150)